data_handler/handlers/loan_states/zklend/settings.py

```diff
@@ -20,70 +22,68 @@ class TokenSettings(ZkLendSpecificTokenSettings, TokenSettings):
-        liquidation_bonus=decimal.Decimal("0.10"),
-        protocol_token_address="0x047ad51726d891f972e74e4ad858a261b43869f7126ce7436ee0b2529a98f486",
-    ),
-    "DAI": ZkLendSpecificTokenSettings(
-        collateral_factor=decimal.Decimal("0.70"),
-        debt_factor=decimal.Decimal("1"),
-        liquidation_bonus=decimal.Decimal("0.10"),
-        protocol_token_address="0x062fa7afe1ca2992f8d8015385a279f49fad36299754fb1e9866f4f052289376",
-    ),
-    "USDT": ZkLendSpecificTokenSettings(
-        collateral_factor=decimal.Decimal("0.80"),
-        debt_factor=decimal.Decimal("1"),
-        liquidation_bonus=decimal.Decimal("0.10"),
-        protocol_token_address="0x00811d8da5dc8a2206ea7fd0b28627c2d77280a515126e62baa4d78e22714c4a",
-    ),
-    "wstETH": ZkLendSpecificTokenSettings(
-        collateral_factor=decimal.Decimal("0.80"),
-        debt_factor=decimal.Decimal("1"),
-        liquidation_bonus=decimal.Decimal("0.10"),
-        protocol_token_address="0x0536aa7e01ecc0235ca3e29da7b5ad5b12cb881e29034d87a4290edbb20b7c28",
-    ),
-    # TODO: Add LORDS.
-    "LORDS": ZkLendSpecificTokenSettings(
-        collateral_factor=decimal.Decimal("1"),
-        debt_factor=decimal.Decimal("1"),
-        liquidation_bonus=decimal.Decimal("0"),
-        protocol_token_address="",
-    ),
-    # TODO: Update STRK settings.
-    "STRK": ZkLendSpecificTokenSettings(
-        collateral_factor=decimal.Decimal("0.50"),
-        debt_factor=decimal.Decimal("1"),
-        liquidation_bonus=decimal.Decimal("0.15"),
-        protocol_token_address="0x06d8fa671ef84f791b7f601fa79fea8f6ceb70b5fa84189e3159d532162efc21",
-    ),
-}
+def convert_string_to_decimal_1e27(string_big_num):
+    """
+    Method to convert big string number to decimal.
+    Example:
+    string_big_num = "800000000000000000000000000"
+    return = 0.80
+    """
+    decimal_big_num = decimal.Decimal(string_big_num)
+    scale_factor = decimal.Decimal("1e27")
+    bigResult = decimal_big_num / scale_factor
+    # Have to cast "0.800000000000000000000000000" to "800000000000000000000000000"
+    formatted_number = "{:.2f}".format(bigResult)
+    return formatted_number
 
-TOKEN_SETTINGS: dict[str, TokenSettings] = {
-    token: TokenSettings(
-        symbol=TOKEN_SETTINGS[token].symbol,
-        decimal_factor=TOKEN_SETTINGS[token].decimal_factor,
-        address=TOKEN_SETTINGS[token].address,
-        collateral_factor=ZKLEND_SPECIFIC_TOKEN_SETTINGS[token].collateral_factor,
-        debt_factor=ZKLEND_SPECIFIC_TOKEN_SETTINGS[token].debt_factor,
-        liquidation_bonus=ZKLEND_SPECIFIC_TOKEN_SETTINGS[token].liquidation_bonus,
-        protocol_token_address=ZKLEND_SPECIFIC_TOKEN_SETTINGS[
-            token
-        ].protocol_token_address,
+def get_value_by_name(data_list, name):
+    """
+    Method to return a specific value in a list based on the name.
+    """
+    return next((item['value'] for item in data_list if item['name'] == name), None)
+
```

Comment:
Split individual functions/objects by two empty lines.

Reply:
I cant separate because the next() function works on that way

**How It Works**

- The function uses a generator expression within the next() function.
- The generator expression iterates over each dictionary in data_list and checks if the 'name' key of the dictionary matches the input name.
- If a match is found, the generator expression yields the corresponding 'value'.
- The next() function returns the first yielded value.
- If no match is found, next() uses the default value None.

Reply:
No, I'm referring to functions like `get_value_by_name`, `get_token_settings`, ..

Reply:
Fix it in this commit: 73f712a53242c2402dbad84e335c8b84ae1e3ebf please let me know
 I separate all the methods using 2 empty spaces